### PR TITLE
Firestore

### PR DESCRIPTION
**Pull request by lindacnatt**
Change database to firestore instead of Realtime database

### CODE REVIEW

**Comment by lindacnatt** [comment]
Bör man hämta datan i Viewmodel:en eller en fil som heter Repository? @johannesloor

File: Halfway/Page Views/UserProfileView.swift [hunk 1]
```diff
@@ -114,6 +116,9 @@ struct UserProfileView: View {
         image = Image(uiImage: inputImage)
         profilepic.emojipic = ""
         
+    }
+    func addUser(Name: String){
+        
     }
 }
```
Comment (johannesloor) [inline Halfway/Page Views/UserProfileView.swift:123]: Empty, to be used later?

**Review by johannesloor — APPROVED** [review]
Did not look at all the pod files but otherwise gooood jooob

File: Halfway/Page Views/UserProfileView.swift (hunk 1 above)
Comment (lindacnatt) [reply Halfway/Page Views/UserProfileView.swift:123]: Yes

**Pull request merged**